cartographer/mapping_2d/sparse_pose_graph.h

```diff
@@ -102,6 +102,13 @@ class SparsePoseGraph : public mapping::SparsePoseGraph {
   mapping::MapById<mapping::NodeId, mapping::TrajectoryNode>
   GetTrajectoryNodes() override EXCLUDES(mutex_);
   std::vector<Constraint> constraints() override EXCLUDES(mutex_);
+  void SetInitialTrajectoryPose(int from_trajectory_id, int to_trajectory_id,
+                                const transform::Rigid3d& pose,
+                                const common::Time& time) override
```

Comment:
Consistency in the codebase: `const common::Time time`? Similar in other places.

Reply:
Done